### PR TITLE
global: fixes on detail view

### PR DESCRIPTION
**Pull request by drjova**

### CODE REVIEW

File: src/literature/components/ReferenceItem.jsx [hunk 1]
```diff
@@ -5,24 +5,25 @@ import { List } from 'antd';
...
 import Latex from '../../common/components/Latex';
-import ArxivEprintList from './ArxivEprintList';
 
 class ReferenceItem extends Component {
   static renderTitle(reference) {
     const recordId = reference.get('control_number');
     const title = reference.getIn(['titles', 0, 'title']);
+    let label = reference.get('label');
+
+    label = label ? `[${label}] ` : '';
```
Comment (harunurhan) [inline src/literature/components/ReferenceItem.jsx]: you can use `.get('label', '')` instead of `?`. Also I think reassign is not very good thing to do.

Comment (drjova) [reply src/literature/components/ReferenceItem.jsx]: that's a good one :)

File: src/common/components/InlineList/InlineList.jsx [hunk 2]
```diff
@@ -40,12 +41,14 @@ InlineList.propTypes = {
   suffix: PropTypes.node,
   label: PropTypes.string,
   separateItems: PropTypes.bool,
+  wrapperClass: PropTypes.string,
```
Comment (harunurhan) [inline src/common/components/InlineList/InlineList.jsx]: `wrapperClassName` 😄

File: src/common/components/InlineList/__tests__/InlineList.test.jsx [hunk 3]
```diff
@@ -25,6 +25,18 @@ describe('InlineList', () => {
     expect(wrapper).toMatchSnapshot();
   });
 
+  it('renderswrapper inline', () => {
```
[the remark this answers is not in the document]
Comment (drjova) [reply src/common/components/InlineList/__tests__/InlineList.test.jsx]: there was a typo

File: src/literature/components/ReferenceItem.jsx [hunk 4]
```diff
@@ -5,24 +5,24 @@ import { List } from 'antd';
...
-import ArxivEprintList from './ArxivEprintList';
 
 class ReferenceItem extends Component {
   static renderTitle(reference) {
     const recordId = reference.get('control_number');
-    const title = reference.getIn(['titles', 0, 'title']);
+    const title = reference.getIn(['titles', 0, 'title'], '');
+    const label = reference.get('label', '');
+
+    const labelDisplay = label ? `[${label}] ` : '';
```
[the remark this answers is not in the document]
Comment (drjova) [reply src/literature/components/ReferenceItem.jsx]: it's by design, we don't want empty labels e.x. ``[]``

File: src/common/components/InlineList/__tests__/__snapshots__/InlineList.test.jsx.snap [hunk 5]
```diff
@@ -140,3 +140,28 @@ exports[`InlineList renders with all props set 1`] = `
   </span>
 </div>
 `;
+
+exports[`InlineList renderswrapper inline 1`] = `
+<div
+  className="__InlineList__"
```
[the remark this answers is not in the document]
Comment (drjova) [reply src/common/components/InlineList/__tests__/__snapshots__/InlineList.test.jsx.snap]: there was a typo

File: src/literature/components/ReferenceItem.jsx (hunk 4 above)
[the remark this answers is not in the document]
Comment (harunurhan) [reply src/literature/components/ReferenceItem.jsx]: ah! right.

**Pull request merged**